tidb-cloud/recovery-group-overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Recovery Group Overview (Beta)
+summary: Learn how to protect your databases against disasters by using TiDB Cloud Recovery Groups.
+---
+
+# Recovery Group Overview (Beta)
+
+TiDB Cloud Recovery Groups allow you to replicate your databases between TiDB Cloud Dedicated clusters for protection against regional disasters. Recovery Groups allow you to orchestrate the failover of databases from one cluster to another. After failover to the secondary cluster, if the original cluster becomes available again, you can re-establish replication in the reverse direction to reprotect the databases.
+
+## Architecture
+
+A Recovery Group consists of a set of replicated databases that can be failed over together between two TiDB Dedicated Clusters. Each Recovery Group is assigned a primary cluster, databases on this primary cluster are associated with the group and are then replicated to the secondary cluster.
+
+![Recovery Group](/media/tidb-cloud/recovery-group/recovery-group-overview.png)
+
+- Recovery Group: a group of databases that are replicated between two clusters
+- Primary Cluster: the cluster where the database is actively written by the application
+- Secondary Cluster: the cluster where replicas of the database are located
```

Comment:
I think we should remind users that we do not guarantee the readonly of replica databases on secondary, and users need to make ensure they won't write to replica databases on secondary.

Reply:
Addressed in https://github.com/pingcap/docs/pull/17425/commits/da21f7502dd8af02816368bae7e09f501b693fa1